rosdep/python.yaml

```diff
@@ -6028,6 +6034,11 @@ python3-pkg-resources:
 python3-prometheus-client:
   debian: [python3-prometheus-client]
   ubuntu: [python3-prometheus-client]
+python3-protobuf:
```

Comment:
Is there a reason this doesn't have a Fedora?

Reply:
@dirk-thomas I added the fedora entry.